src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToHtml5LocalDateTimeTransformer.php

```diff
@@ -54,7 +67,7 @@ public function transform(mixed $dateTime): string
             $dateTime = $dateTime->setTimezone(new \DateTimeZone($this->outputTimezone));
         }
 
-        return $dateTime->format(self::HTML5_FORMAT);
+        return $dateTime->format($this->withSeconds ? self::HTML5_FORMAT : self::HTML5_WITHOUT_SECONDS_FORMAT);
     }
 
     /**
```

Comment:
Reverse transform already works without seconds. We can't "remove" the seconds from a \DateTime instance so there's nothing else to do